d4s/src/main/scala/d4s/models/conditions/Condition.scala

```diff
@@ -126,6 +126,15 @@ object Condition {
     }
   }
 
+  final case class attribute_is_null(path: List[String]) extends Condition.Direct {
+    override protected def eval(nesting: Int): FinalCondition = {
+      val (alias, map) = createAlias(path)
+      val attrValues   = Map(":null" -> AttributeValue.builder().nul(true).build())
+      val condExpr     = s"$alias = :null OR attribute_not_exists($alias)"
```

Comment:
@YaroslavSahach I think we should check only on null condition here. Circe encoding style for options is not implemented yet.

//cc @neko-kai @vpodlnk

Reply:
yes, we should check only for `NULL`, since the current `option type` encoder will set an attribute value to `null` if it equals to `None` in `Scala`
also, IMO, it'll be a bit confusing to check on existence, because it is kinda a different assertion 🤷🏻‍♂️